module/zfs/zfs_sysfs.c

```diff
@@ -356,38 +356,68 @@ pool_property_show(struct kobject *kobj, struct attribute *attr, char *buf)
  * async calls (i.e. a sync flag). If this mechanism were in place at that
  * time, we could have added a 'channel_program_async' to this list.
  */
-static const char *zfs_features[]  = {
-	/* --> Add new kernel features here (post ZoL 0.8.0) */
-	"initialize",
-	"trim",
+static const char *zfs_kernel_features[]  = {
+	/* --> Add new kernel features here */
+	"com.delphix.vdev_initialize",
+	"org.zfsonlinux.vdev_trim",
```

Comment:
I believe you meant to use a `:` here instead of a `.` before the feature name.  This would be consistent with `features.pool`.

/sys/module/zfs/features.kernel/
```
	"com.delphix:vdev_initialize",
	"org.zfsonlinux:vdev_trim",
```

/sys/module/zfs/features.pool/
```
	"com.delphix:device_removal"
```

Reply:
Yep, good catch.